config.py

```diff
@@ -9,7 +9,15 @@
 SQLALCHEMY_DATABASE_URI = 'postgresql://{0}:{1}@{2}:{3}/{4}'.format(
     DB_USER, DB_PASS, DB_SERVICE, DB_PORT, DB_NAME
 )
-DEBUG_TB_INTERCEPT_REDIRECTS = False
-DEBUG_TB_PROFILER_ENABLED = True
-SQLALCHEMY_TRACK_MODIFICATIONS = False
+# any value passed through env var will make them
+# True (we don't check the content of the variables)
+DEBUG_TB_INTERCEPT_REDIRECTS = (
+    os.environ.get("DEBUG_TB_INTERCEPT_REDIRECTS", None) is not None
+)
```

Comment:
The problem here is that any value for the environment variable will be considered as `True`.

We may need something like:
``` python
def str_to_bool(value):
    return str(value).lower() in {"true", "1", "t", "yes", "y"}
```

Reply:
i know; i have done a quick research on the python stdlib... without success :-( We can at least write our own function for that, but i don't like the idea to have that  kind of stuff into something like `config.py`.

Reply:
done!